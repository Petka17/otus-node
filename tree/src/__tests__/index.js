import { stringifyTree } from '../index.js'

test('tree stringigy work as expected', () => {
  const tree = {
    name: 1,
    items: [
      {
        name: 2,
        items: [
          {
            name: 3,
          },
          {
            name: 4,
            items: [
              {
                name: 9,
              },
            ],
          },
          {
            name: 7,
          },
        ],
      },
      {
        name: 5,
        items: [
          {
            name: 6,
          },
        ],
      },
    ],
  }

  expect(stringifyTree(tree)).toMatchSnapshot()
})

