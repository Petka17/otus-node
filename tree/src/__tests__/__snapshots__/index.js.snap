// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tree stringigy work as expected 1`] = `
"1
├-2
| ├-3
| ├-4
| | └-9
| └-7
└-5
  └-6"
`;
